### ISSUE TITLE
Audio Mixing Feature

### ISSUE THREAD

**Issue by poudink** [ajusa/VNDS-LOVE#22]
**Is your feature request related to a problem? Please describe.**
Missing feature from the original VNDS software.

**Describe the solution you'd like**
A menu that allows changing the volume of SFX and music. On the DS, the volume for both could be adjusted separately and you could choose any value between 0 (silence) and 16 (max volume).

**Comment by ajusa** [comment]
Should this be a globally saved setting? Eg, you set music to be 15 and sound to be 2 while playing through one novel. Once you exit and open a different novel, should it keep the same settings? Or should each novel have its own settings?
Personally, global settings would be way easier and probably have fewer issues. 
Note that I'll probably need to handle this the same way I would handle any other (future) settings, such as font size, number of lines to show, colors, stretch vs fit, etc.
https://github.com/videah/Love_INI_Parser for storing the config file

**Comment by poudink** [comment]
Well it'd be nice to have per-novel settings for this because different VNDS novels have different audio mixing, in the original VNDS software this is a global setting.

**Comment by ajusa** [comment]
This is nearly done with https://github.com/ajusa/VNDS-LOVE/commit/ea08634fc4fbca2480f2bc803e0b00756a3b4b5d

Just need to create an actual menu, but you can set the volume for each kind of audio inside of a config file. I'll put out a new release once that is done.
